translate/en.json

```diff
@@ -26,7 +26,6 @@
   "fastingMessage": "Are you fasting?",
   "mainKeyboard": [
```

Comment:
why we need this file?

Reply:
We don’t need it anymore. I just wanted to keep your hard work)

Reply:
I’ll delete it

Reply:
okay, don't worry my hard work stays on github history

Reply:
fixed